README.md

```diff
@@ -286,11 +286,13 @@ import { registerExtension, initialize } from 'vscode/extensions'
 
 await initialize()
 
-const { registerFile: registerExtensionFile, getApi } = registerExtension(defaultThemesExtensions)
+const { registerFileUrl: registerExtensionFile, getApi } = registerExtension(defaultThemesExtensions)
 
-registerExtensionFile('/file.json', async () => fileContent)
+registerFileUrl('/file-extension-path.json', new URL('./file-real-path.json', import.meta.url))
```

Comment:
I think this may still be incorrect, or at least it doesn't demonstrate loading the extension (which as best as I can tell is done via `registerExtensionFile` from `@codingame/monaco-vscode-files-service-override`).

Reply:
`defaultThemesExtensions` is misleading maybe. @CGNonofr and @brianjenkins94 What about:
```
const extension = {
    name: 'my-extension',
    publisher: 'someone',
    version: '1.0.0',
    engines: {
        vscode: '*'
    },
    contributes: {
    }
};
const { registerFileUrl } = registerExtension(extension, ExtensionHostKind.LocalProcess);
```

Reply:
And that's it? There's no additional step to register/load it?

Reply:
@brianjenkins94 you have to use `registerFileUrl`, of course. I updated the code snippet above ⬆️

Reply:
@brianjenkins94 Here you find a full example: https://github.com/TypeFox/monaco-languageclient/blob/main/packages/examples/src/langium/statemachineClient.ts

@CGNonofr do you want to change the README once more?

Reply:
indeed it was still wrong, commit amended